.gitignore

```diff
@@ -0,0 +1,10 @@
+**/*.in
+**/*.o
```

Comment:
I think `*.o` is enough if you want to ignore all `.o` files in the repo. The same to the other lines in the file.

Reply:
Ok.